### ISSUE TITLE
pytorch转模型失败

### ISSUE THREAD

**Issue by alexiycv** [PaddlePaddle/X2Paddle#827]
The unsupported packages or operators are: [torch.nn.MSELoss].
The unsupported packages or operators are: [torch.nn.Linear_block, torch.nn.functional.normalize].
The unsupported packages or operators are: [torch.is_tensor, torch.ByteStorage.from_buffer].

**Comment by wjj19950828** [comment]
@Bruno-bai 您好，感谢反馈～

看到有相关loss计算，如果是预测的话能否把loss计算去掉呢？

去掉后还有问题的话，麻烦提供一下相关模型以及转换脚本

另外还有两个问题需要描述一下：
1、具体业务场景
2、为什么有转到Paddle部署的需求呢？

感谢～
